openassessment/assessment/api/peer.py

```diff
@@ -45,7 +45,7 @@ def submitter_is_finished(submission_uuid, requirements):
         bool
 
     """
-    if not requirements:
+    if requirements is None:
```

Comment:
I think both `peer` and `student_training` should at least be handled consistently, yeah?